.github/workflows/ci.yaml

```diff
@@ -29,8 +29,22 @@ on:
       - ".gitmodules"
       - ".gitattributes"
       - ".editorconfig"
-
 jobs:
+  lint:
```

Comment:
I think it is a decent enough idea to make this its own job, but can we make it run on python "3.10" and then remove it from the individual test runs:  https://github.com/pypa/pipenv/pull/5059/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR81

Reply:
Was working on that while you wrote the comment ...